shared/devices/existing-device/index.js

```diff
@@ -14,22 +15,11 @@ type Props = {
 
 class ExistingDevice extends Component<void, Props, void> {
   render () {
-    let otherDeviceComputer = null
-    let otherDevicePhone = null
+    const otherRoleIsNew = this.props.myDeviceRole === codePageDeviceRoleExistingPhone ||
```

Comment:
This refactor was required cause flow was confused about the state of the variables. it complained that they might be null but the switch covered all cases and had a default so i changed it to a different for so it'd not complain

@chromakode (wanted comments on the original $FlowIssue, but he's on vacation)